src/sbvr-api/errors.ts

```diff
@@ -8,10 +10,7 @@ export class SbvrValidationError extends TypedError {}
 
 export class TranslationError extends TypedError {}
 export class ParsingError extends TypedError {}
-
-export interface AnyObject {
-	[index: string]: any;
-}
+export { AnyObject };
```

Comment:
I think it's best to not export this one, getting it from common-types everywhere is better IMO.

Reply:
This is necessary for backwards compatibility, I'll drop this in the major